apps/web/public/locales/de/common.json

```diff
@@ -0,0 +1,72 @@
+{
+  "addition-date": "Hinzufügungsdatum",
+  "aria": {
+    "students-union-logo": "Studenten-Union-Logo",
+    "toggle-navigation": "Navigation umschalten",
+    "user-menu": "Benutzer-Menü"
+  },
+  "download": "downloaden",
+  "Download": "Download",
+  "error": {
+    "change-language-error": "Fehler beim Ändern der Sprache",
+    "title": "Fehler"
+  },
+  "first-name": "Vorname",
+  "functions": "Funktionen",
+  "language-code": "de",
+  "languages": {
+    "de": {
+      "flag": "🇩🇪",
+      "name": "Deutsch"
+    },
+    "en": {
+      "flag": "🇬🇧",
+      "name": "Englisch"
+    },
+    "pl": {
+      "flag": "🇵🇱",
+      "name": "polski"
```

Comment:
Capitalize for consistency

Reply:
Nope, look into another pages - in Poland we use not capitalized letters. I've seen this in many games or websites